pkgs/development/python-modules/kivy/default.nix

```
@@ -0,0 +1,60 @@
+{ lib
+, buildPythonPackage, fetchPypi, fetchpatch
+, pkg-config, cython
+, kivy-garden, requests, docutils, pygments
+, mesa, mtdev, SDL2, SDL2_image, SDL2_ttf, SDL2_mixer, gst_all_1
+, pillow
+}:
+
+buildPythonPackage rec {
+  pname = "Kivy";
+  version = "2.0.0rc4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1l19kfsprgrh2a19s11zbwgjdgbwh4lvnwg27rri01kgvsisbhqn";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    cython
+  ];
+
+  buildInputs = [
+    kivy-garden
+    requests
+    docutils
+    pygments
+
+    mesa
```

```suggestion
    mesa
```
there's a python package of the same name, but probably not what you want

I would probably explicitly bring these in from the pkgs scope. see other comment

---

Yep indeed. Fixed.